jobserver/api/releases.py

```diff
@@ -320,6 +324,35 @@ def post(self, request, release_id):
 
         slacks.notify_release_file_uploaded(rfile)
 
+        # if we're dealing with an HTML file and the name matches an
+        # AnalysisRequest PK create a report for it
+        name, ext = os.path.splitext(filename)
+        if ext == ".html":
+            # TODO: this try block exists to handle TimeflakePrimaryKeyBinary raising
```

Comment:
Is it worth moving the contents of this if-statement into a separate function? It might be a bit distracting if you're looking at this and don't care about Analysis Requests and Reports.